docs/Developers/testnet.md

```diff
@@ -454,3 +454,110 @@ SelfKisser_1 | [0xc0fe3a070Bc98b4a45d735A52a1AFDd134E0283f](https://sepolia.arbi
 | Chronicle_ZK_USDT_1        | [0xF3A5fE18CD97fd2d828bE105245fF3Fe24ADA93b](https://bartio.beratrail.io//address/0xF3A5fE18CD97fd2d828bE105245fF3Fe24ADA93b) |
 
 </details>
+
+<details>
+<summary> Smart Contract Addresses on **Corn Testnet** </summary>
+
+| Contract Name              | Contract Address on Corn Testnet                                                                                           |
+| -------------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| SelfKisser_1               | [0x8CA2E3a09c79233127dC21955e28A6b9C4bf166E](https://testnet.cornscan.io/address/0x8CA2E3a09c79233127dC21955e28A6b9C4bf166E) |
+ |Chronicle_BTC_USD_1 |  [0x152598809FB59db55cA76f89a192Fb23555531D8](https://testnet.cornscan.io/address/0x152598809FB59db55cA76f89a192Fb23555531D8)
```

Comment:
nit: In case you use VSCode I would recommend a "table prettifier" plugin like https://marketplace.visualstudio.com/items?itemName=darkriszty.markdown-table-prettify.

Reply:
Thank you, just installed it now 🫡